controls/V-71999.rb

```diff
@@ -55,14 +55,16 @@
   tag "fix": "Install the operating system patches or updated packages
 available from Red Hat within 30 days or sooner as local policy dictates."
   tag "fix_id": "F-78351r1_fix"
-  describe.one
-    describe "All packages are up to date" do
-      subject { linux_update.updates.length }
-      it { should eq 0 }
+  describe.one do
+    describe 'List of out-of-date packages' do
+      subject { linux_update.updates }
+      it { should be_empty }
```

Comment:
What was the driver here as well? More for my own knowlege than anything else...